pkgs/development/python-modules/pychart/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy27 }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.

Reply:
Fixed :+1: